public/app/core/components/AppChrome/TopBar/SingleTopBar.tsx

```diff
@@ -118,7 +118,17 @@ const getStyles = (theme: GrafanaTheme2, menuDockedAndOpen: boolean) => ({
       gridTemplateColumns: '2fr minmax(550px, 1fr)',
       display: 'grid',
       justifyContent: 'flex-start',
+      ...(menuDockedAndOpen && {
```

Comment:
Not sure if these are leftover from the previous PR, but dont think it is related.